std/src/bin/rgb-stl.rs

```diff
@@ -80,7 +95,7 @@ fn main() {
             Some(
                 "
   Description: RGB standard library
-  Author: Dr Maxim Orlovsky <orlovsky@lnp-bp.org>
+  Author: Zoe Faltibà
```

Comment:
You missed the line, this should go above to RGB-25 :D

Reply:
```suggestion
  Author: Dr Maxim Orlovsky <orlovsky@lnp-bp.org>
```

Reply:
oops sorry :) going to fix it